Jenkinsfile

```diff
@@ -1106,6 +958,69 @@ pipeline {
                 expression { ! commitPragma(pragma: 'Skip-test').contains('true') }
             }
             parallel {
+                stage('Coverity on CentOS 7') {
+                    // Eventually this will only run on Master builds.
+                    // Unfortunately for now, a PR build could break
+                    // the quickbuild, which would not be detected until
+                    // the master build fails.
+//                    when {
+//                        beforeAgent true
+//                        anyOf {
+//                            branch 'master'
+//                            not {
+//                                // expression returns false on grep match
+//                                expression {
+//                                    sh script: 'git show -s --format=%B |' +
+//                                               ' grep "^Coverity-test: true"',
+//                                    returnStatus: true
+//                                }
+//                            }
+//                        }
+//                    }
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.centos.7'
+                            dir 'utils/docker'
+                            label 'docker_runner'
+                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " +
+                                                '$BUILDARGS ' +
+                                                '--build-arg QUICKBUILD=0' +
+                                                ' --build-arg REPOS="' + cart_repos + '"'
+                        }
+                    }
+                    steps {
+                        sh "rm -f coverity/daos_coverity.tgz"
```

Comment:
shouldnt this be cart_coverity.tgz based on line 1004?

Reply:
Yes.  will fix in the next commit.